pose-detection/src/blazepose_tfjs/detector.ts

```diff
@@ -389,10 +419,39 @@ class BlazePoseTfjsDetector implements PoseDetector {
         constants.BLAZEPOSE_NUM_KEYPOINTS,
         constants.BLAZEPOSE_NUM_AUXILIARY_KEYPOINTS);
 
+    // -------------------------------------------------------------------------
+    // ------------------------- Pose world landmarks --------------------------
+    // -------------------------------------------------------------------------
+
+    // Decodes the world landmark tensors into a list of landmarks, where the
+    // landmark coordinates are normalized by the size of the input image to the
+    // model.
+    // PoseLandmarksByRoiCpu: TensorsToLandmarksCalculator.
+    const worldLandmarks = await tensorsToLandmarks(
```

Comment:
World landmarks shouldn't be normalized by the image size and your comment states that they are (although not sure what is happening inside the method). Please update comment and check that nothing happens to world landmarks in the method (i.e. returned as they are from the model).

Reply:
This change was temporary and is now gone in the newer commits